diff --git a/node_modules/redoc-cli/index.js b/node_modules/redoc-cli/index.js
index 0164f15..9005774 100755
--- a/node_modules/redoc-cli/index.js
+++ b/node_modules/redoc-cli/index.js
@@ -294,7 +294,7 @@ function getPageHTML(spec, pathToSpec, { ssr, cdn, title, disableGoogleFont, tem
                 : '<script src="redoc.standalone.js"></script>',
             title: title || spec.info.title || 'ReDoc documentation',
             disableGoogleFont,
-            templateOptions,
+            ...templateOptions,
         });
     });
 }
